src/testController.ts

```diff
@@ -142,28 +142,26 @@ export class TestController {
         continue;
       }
 
-      if (test.id.startsWith("test_")) {
```

Comment:
Wasn't this check here to disambiguate minitest/test_unit tests from other kinds of tests, like RSpec tests? If so, we should make sure we mark tests properly as minitest/test_unit/rspec etc to be able to check appropriately here before removing this. Otherwise the output parsing code below might fail in a really bad way.

Reply:
I'm made an update which pass that additional information, and then uses tags to determine the behaviour. The problem is that this would break for older versions of the gem, so I'm wondering if we should [check for a minimum version](https://github.com/Shopify/ruby-dev-exp-issues/issues/921) of the gem.

cc @shopify-aiden, @bitwise-aiden

Reply:
Checking for a minimum version makes a lot of sense to me. We want to make sure to keep the two in sync in the long run, so adding this feature now will help make steps toward that 👍🏻